package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Resolve AMD module names to File objects suitable to run in the Browser",
   "main": "dist/amd-resolver.js",
   "scripts": {
-    "prepublish": "browserify src/resolver.js -o dist/amd-resolver.js -s amd-resolver --dg false",
+    "prepublish": "grunt compile-production",
```

Comment:
Grunt cli is installed prior to running npm tasks right [here](https://github.com/MiguelCastillo/amd-resolver/blob/master/.travis.yml#L6).  So that grunt should not be the problem.  The problem is that npm:prepublish is calling `compile-production`, which no longer exists.